README.md

```diff
@@ -1,3 +1,5 @@
 # Silverstripe CMS roadmap and support timeline JSON
 
-Holds JSON data used for the Silverstripe CMS roadmap and support timeline
+Holds data.son used for the Silverstripe CMS roadmap and support timeline
+
+The other files in the repository are for validating that data.json is valid in ci
```

Comment:
Please add some information about what we need to update here when we do a release.

Reply:
Updated